docs/sources/configuration/_index.md

```diff
@@ -196,6 +196,28 @@ configures the HTTP and gRPC server communication of the launched service(s).
 # CLI flag: -server.http-listen-port
 [http_listen_port: <int> | default = 80]
 
+# TLS configuration for serving over HTTPS
+http_tls_config:
+  # HTTP server cert path.
+  # CLI flag: -server.http-tls-cert-path
+  [cert_file: <string> | default = ""]
+  # HTTP server key path.
+  # CLI flag: -server.http-tls-key-path
+  [key_file: <string> | default = ""]
+  # HTTP TLS Client Auth type.
+  # CLI flag: -server.http-tls-client-auth
+  # Possible values:
+  # - RequestClientCert
+  # - RequireAnyClientCert
+  # - RequireClientCert
+  # - VerifyClientCertIfGiven
+  # - RequireAndVerifyClientCert
+  # - "" (Empty string) or "NoClientCert"
```

Comment:
No need to add these possible values. As this is related to the weaveworks library 
```suggestion
```

Reply:
I removed the suggested lines in a new commit.